t/discovery/nacos3.t

```diff
@@ -0,0 +1,636 @@
+--- extra_yaml_config
+discovery:
+  nacos:
+    host:
+      - "http://127.0.0.1:8858"
+    fetch_interval: 1
+    access_key: "my_access_key"
+    secret_key: "my_secret_key"
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+
+           -- use nacos-service5
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "uri": "/hello",
+                    "upstream": {
+                        "service_name": "APISIX-NACOS",
+                        "discovery_type": "nacos",
+                        "type": "roundrobin",
+                        "discovery_args": {
+                          "namespace_id": "test_ns",
+                          "group_name": "test_group"
+                        }
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+
+            -- use nacos-service6
+            local code, body = t('/apisix/admin/routes/2',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "uri": "/hello1",
+                    "upstream": {
+                        "service_name": "APISIX-NACOS",
+                        "discovery_type": "nacos",
+                        "type": "roundrobin",
+                        "discovery_args": {
+                          "namespace_id": "test_ns",
+                          "group_name": "test_group2"
+                        }
+                    },
+                    "plugins": {
+                        "proxy-rewrite": {
+                            "uri": "/hello"
+                        }
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+
+            ngx.sleep(1.5)
+
+            local http = require "resty.http"
+            local httpc = http.new()
+            local uri1 = "http://127.0.0.1:" .. ngx.var.server_port .. "/hello"
+            local res, err = httpc:request_uri(uri1, { method = "GET"})
+            if err then
+                ngx.log(ngx.ERR, err)
+                ngx.status = res.status
+                return
+            end
+            ngx.say(res.body)
+
+            local uri2 = "http://127.0.0.1:" .. ngx.var.server_port .. "/hello1"
+            res, err = httpc:request_uri(uri2, { method = "GET"})
+            if err then
+                ngx.log(ngx.ERR, err)
+                ngx.status = res.status
+                return
+            end
+            ngx.say(res.body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+server 1
+server 3
+
+
+
+=== TEST 17: same group_name and service_name, different namespace_id
+--- extra_yaml_config
+discovery:
+  nacos:
+    host:
+      - "http://127.0.0.1:8858"
+    fetch_interval: 1
+    access_key: "my_access_key"
+    secret_key: "my_secret_key"
```

Comment:
If these configurations don't work, why do you add so many test cases?

Reply:
because we can use mse nacos instead of it and run test in local environment easily, and also see does it take any effect in open sourece verison. what do you think of it?

Reply:
Could you add a comment about this, like: access_key and secret_key won't work and affect the open source nacos